crates/subspace-farmer/src/bin/subspace-farmer/main.rs

```diff
@@ -287,6 +296,26 @@ async fn main() -> Result<()> {
                 command.farm
             };
 
+            if farming_args.piece_publisher_batch_size == 0
+                || farming_args.piece_publisher_batch_size > MAX_PIECE_PUBLISHER_BATCH_SIZE
+            {
+                return Err(anyhow::anyhow!(
+                    "Incorrect piece publisher batch size: {}. Should be 1-{}",
```

Comment:
Could be more Rusty, though for regular users may not make much sense
```suggestion
                    "Incorrect piece publisher batch size: {}. Should be 1..={}",
```

Reply:
IMHO, non-rusters 99,999999% could be confused 😃